src/foraging_gui/bias_indicator.py

```diff
@@ -231,6 +231,8 @@ def clear(self):
         # reset bias list
         self._biases = []
         # reset upper and lower ci
+        self._upper_scatter_item = PlotDataItem([0], [0], pen='lightgray')
```

Comment:
Can you make a note on the pull request about adding this part? Or you can remove this part here

Reply:
I'll make a note!